applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -145,7 +145,7 @@ def _check_outputs(self,mp,A,dim):
         E = mp.GetProperties()[1].GetValue(KratosMultiphysics.YOUNG_MODULUS)
         NU =mp.GetProperties()[1].GetValue(KratosMultiphysics.POISSON_RATIO)
         
-        #given the matrix A, the analytic deformation graident is F+I
+        #given the matrix A, the analytic deformation gradient is F+I
         F = A
         for i in range(3):
```

Comment:
Yes F = DF*F0, but to use the interface from the base solid I call F as FT (total F) and DF as F (because is computed at the same time that others elements)

Do I reopen this?